docs/contributing.rst

```diff
@@ -80,10 +80,8 @@ Finally, please make sure you respect the `coding style
 for this project. Also, even though we do CI testing, please test your code and
 ensure that it builds locally before submitting a pull request.
 
-We highly recommend going through our `review checklist <https://github.com/ethereum/solidity/blob/develop/ReviewChecklist.md>`
-before submitting the pull request.
-We thoroughly review every PR and will help you get it right, but there are many
-common problems that can be easily avoided, making the review much smoother.
+We highly recommend going through our `review checklist <https://github.com/ethereum/solidity/blob/develop/ReviewChecklist.md>`_ before submitting the pull request. 
```

Comment:
```suggestion
We highly recommend going through our `review checklist <https://github.com/ethereum/solidity/blob/develop/ReviewChecklist.md>`_ before submitting the pull request.
```

Build is failing due to the trailing whitespace, so you're gonna have to fix it before we merge.